docs/headers.md

```diff
@@ -134,7 +134,7 @@ constexpr byte operator~(byte b) noexcept;
 Bitwise negation of a `byte`. Flips all bits. Zeroes become ones, ones become zeroes.
 
 ```cpp
-template <class IntegerType, class = std::enable_if_t<std::is_integral<IntegerType>::value>>
+template <class IntegerType, std::enable_if_t<std::is_integral<IntegerType>::value, bool> = true>
```

Comment:
This is a change I am willing to accept because it makes the following example not compile:
```c++
#define GSL_USE_STD_BYTE 0
#include <gsl/gsl>
static_assert(std::is_floating_point_v<decltype(to_integer<float, float>(gsl::byte(0)))>, "oof");
```
(which [currently compiles](https://godbolt.org/z/1efd7sxsa) on `main`)